lib/jblond/Diff.php

```diff
@@ -221,6 +225,20 @@ public function getArrayRange(array $array, int $start = 0, $end = null): array
         return array_slice($array, $start, $length);
     }
 
+    /**
+     * Get if the compared versions are identical or have differences.
+     *
+     * @return bool True when identical.
+     */
+    public function isIdentical(): bool
+    {
+        if ($this->getGroupedOpCodes() === null) {
+            $this->getGroupedOpCodes();
```

Comment:
Why call it twice? In the if statement it is already called. Or do I miss something?

Reply:
Good catch... I think phpstorm was being to helpful on this one. 😅